File: app/src/main/java/br/com/santander/testeandroid/ui/contact/domain/models/Type.java
package br.com.santander.testeandroid.ui.contact.domain.models;

import com.google.gson.annotations.SerializedName;

public enum Type {
    @SerializedName("1")
    FIELD,
    @SerializedName("2")
    TEXT,
    @SerializedName("3")
    IMAGE,
    @SerializedName("4")
    CHECKBOX,
    @SerializedName("5")
    SEND
}

File: app/src/main/java/com/santander/luizlago/testeandroid/enums/Type.kt
package com.santander.luizlago.testeandroid.enums

enum class Type(val value: Int) {
    FIELD(1),
    TEXT(2),
    IMAGE(3),
    CHECKBOX(4),
    SEND(5);

    companion object {
        fun valueOf(value: Int): Type? = Type.values().find { it.value == value }
    }
}